engine/servermaster/jobmanager.go

```diff
@@ -579,7 +607,12 @@ func (jm *JobManagerImpl) OnWorkerStatusUpdated(worker framework.WorkerHandle, n
 // CloseImpl implements frame.MasterImpl.CloseImpl
 func (jm *JobManagerImpl) CloseImpl(ctx context.Context) error {
 	jm.notifier.Close()
-	return nil
+	return jm.wg.Wait()
+}
+
+// StopImpl implements frame.MasterImpl.StopImpl
```

Comment:
`bgJobOperatorLoop` may call `Exit` and trigger `StopImpl` for the  jobmananger.
Will it be a problem?

Reply:
Yes, changed the implement of `StopImpl` in job manager.